<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputRepeater</class>
 <widget class="QMainWindow" name="InputRepeater">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Input Repeater</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="layoutCentralWidget">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="styleSheet">
       <string notr="true">QLabel {
	font-size: 20pt;
	color: darkred;
}</string>
      </property>
      <property name="text">
       <string>Input Repeater</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="RecordTabWidget" name="recordTabWidget">
       <attribute name="title">
        <string>Recording</string>
       </attribute>
      </widget>
      <widget class="ReplayTabWidget" name="replayTabWidget">
       <attribute name="title">
        <string>Replay</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="InputRepeaterStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRecord"/>
   <addaction name="actionReplay"/>
   <addaction name="actionOptions"/>
   <addaction name="actionAbout"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionRecord">
   <property name="text">
    <string>Record</string>
   </property>
  </action>
  <action name="actionReplay">
   <property name="text">
    <string>Replay</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>RecordTabWidget</class>
   <extends>QWidget</extends>
   <header>RecordTabWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>InputRepeaterStatusBar</class>
   <extends>QStatusBar</extends>
   <header>InputRepeaterStatusBar.hpp</header>
  </customwidget>
  <customwidget>
   <class>ReplayTabWidget</class>
   <extends>QWidget</extends>
   <header>ReplayTabWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MouseKeyboardRecorder.qrc"/>
 </resources>
 <connections/>
</ui>
